packages/firestore/test/unit/specs/spec_builder.ts

```diff
@@ -425,35 +423,38 @@ export class SpecBuilder {
    * expectUserCallback defaults to true if options are omitted.
    */
   writeAcks(
+    docs: string[],
```

Comment:
Making this an array was indeed inspired by your previous comments. I debated adding an overload that just takes a string, but have now gone in favor of just accepting a string for now. It still makes sense to accept multiple callbacks though (after a drainQueue call).

I left the name as `doc` since that name used widely throughout the SpecTest framework.